src/Go/boardAnalysis/goAI.ts

```diff
@@ -783,7 +764,7 @@ function getMoveOptions(
     random: undefined,
   };
 
-  const moveOptionGetters: { [s in keyof MoveOptions]: () => Promise<Move | null> } = {
+  const moveOptionGetters: MoveOptions = {
```

Comment:
I'm pretty sure that you can leave off declaring the type here, and you might not need `MoveOptions` at all. (You can always use `typeof moveOptionsGetters`). But, it's not really hurting anything.